opensoundscape/datasets.py

```diff
@@ -377,3 +380,58 @@ def __getitem__(self, item_idx):
 
         # Return data only (prediction)
         return {"X": X}
+
+
+# class PredictionDataset(torch.utils.data.Dataset):
```

Comment:
Should we keep this in here? (If we might need it later, would be fine to leave it commented)